src/style.css

```diff
@@ -1,3 +1,95 @@
+/* reset browser default margins */
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  color: var(--main-collor) #000;
+  background-color: #fff;
+}
+/* #region header */
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo__link {
+  color: var(--main-collor);
+  text-decoration: none;
+  display: block;
+  padding: 10px 0;
+}
```

Comment:
all these styles are not needed. try to remove them and you won't see any difference, exept of padding, but you can achieve the same behavior with `align-items: center;` for `.header`